Sources/PointFree/Account/Team.swift

```diff
@@ -89,7 +89,7 @@ private func sendEmailsForTeammateRemoval(owner: Database.User, teammate: Databa
   .map(const(unit))
 }
 
-private let teamView = View<([Database.TeamInvite], [Database.User], Database.User, Prelude.Unit)> { invites, teammates, currentUser, _ in
```

Comment:
±1:

- The `Unit` terminator disappears a bunch of places that we previously needed to explicitly destructure/ignore.